### PR TITLE
Testing membership to help for security reasons

### PR DESCRIPTION
**Pull request by cooktheryan**
Wanting to test a way to verify membership to the repo before tests will run

### CODE REVIEW

**Comment by cooktheryan** [comment]
@Gregory-Pereira @lmilbaum can you take a look at this approach?

File: .github/workflows/training-e2e.yaml [hunk 1]
```diff
@@ -27,9 +27,48 @@ concurrency:
...
+
+  check-membership:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if user is a member of the organization
+        id: check_membership
+        run: |
+          ORG="your-organization"
+          USER=${{ github.actor }}
+          RESPONSE=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" "https://api.github.com/orgs/containers/members/$USER")
```
Comment (lmilbaum) [inline .github/workflows/training-e2e.yaml:51]: Would that work if this pipeline triggered from a forked PR? Specifically asking about the secrets.GITHUB_TOKEN.

Comment (lmilbaum) [reply .github/workflows/training-e2e.yaml:51]: Maybe there is a GitHub Actions predefined variable that has this value already populated

**Comment by lmilbaum** [comment]
Would it make sense to develop this on a test repo and then apply it on a production pipeline?

File: .github/workflows/training-e2e.yaml (hunk 1 above)
Comment (cooktheryan) [reply .github/workflows/training-e2e.yaml:51]: so GITHUB_TOKEN should be a predefined variable defined in Github. So usually we do not set these.

**Pull request closed without merging**